### DISCUSSION TITLE
Update to SSL certificates

### DISCUSSION THREAD

**Discussion by langsharpe** [pyrou/traefik.me#6] [General]
Hi @pyrou,

If I'm correct, the current certificate available on traefik.me to download will expire on July 11th (3 days from now). I was wondering if/when you are planning to update this. We use this domain with its certificates and find it really useful (as I'm sure others do) 🙏🏻.

Cheers,

Lang

**Comment by pyrou** [comment]
Certificate renewed 😉

**Comment by langsharpe** [comment]
Hey @pyrou, ready to update again? We really appreciate this service.

**Reply by pyrou** [reply]
Renewed, I feel very sorry. 

I really need to automate this 😢.. and no I can't use certbot out of the box, because I have to use DNS challenge because of wildcard certificate

**Reply by developius** [reply]
Thank you for renewing the certificate @pyrou. 
It looks like cerbot can do DNS-based renewal for wildcard certificates with an extension. DigitalOcean have an article discussing it: https://www.digitalocean.com/community/tutorials/how-to-acquire-a-let-s-encrypt-certificate-using-dns-validation-with-acme-dns-certbot-on-ubuntu-18-04.
Might that work for automating renewal?

**Reply by pyrou** [reply]
> The acme-dns-certbot tool is used to connect Certbot to a third-party DNS server where the certificate validation records can be set automatically via an API when you request a certificate

ALL traefik.me DNS entries are handled by the server itself. I am my own SOA and NS. So the "API" they mention just doesn't exist in my case, (I just need to motivate myself to make this happen)
